_includes/header.html

```diff
@@ -8,3 +10,23 @@ <h1 class='title'><a href="{{ '/' | relative_url }}">INSCIDAR</a></h1>
     </ul>
   </nav>
 </header>
+
+<style>
+  .skip-link {
+  position: absolute;
+  top: -100px;
+  left: 0px;
+  width: 100%;
+  background: #fff;
+  color: #000;
+  padding: 8px 16px;
+  z-index: 100;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: top 0.3s ease;
+}
+
+.skip-link:focus {
+  top: 10px;
+}
+</style>
```

Comment:
Since we also have a separate folder just for styles (e.g., `_sass/base.scss`), should we move this to there? For example, we could move this to `base.scss` or make a file just for the header, like `_sass/header.scss`. For the latter, my understanding is that we need to add an import of the file in `index.scss` ([link](https://github.com/inscidar/inscidar/blob/16b25c5036b4caa1b3450c7466d604033e80e30f/index.scss#L5)):

https://github.com/inscidar/inscidar/blob/16b25c5036b4caa1b3450c7466d604033e80e30f/index.scss#L5

Reply:
```diff
  @import "base";
+ @import "header";
```

Reply:
Ah, it seems like the header is also already in this [here](https://github.com/inscidar/inscidar/blob/16b25c5036b4caa1b3450c7466d604033e80e30f/_sass/base.scss#L118), I can just add the styles all to this base file as well?